website/docs/routes/journal.md

```diff
@@ -411,6 +411,24 @@ In <https://pubmed.ncbi.nlm.nih.gov/trending/?filter=simsearch1.fha&filter=pubt.
 
 </Route>
 
+## Royal Society of Chemistry {#royal-society-of-chemistry}
+
+### Journal {#royal-society-of-chemistry-journal}
+
+<Route author="nczitzk" example="/rsc/journal/ta" path="/rsc/journal/:id/:category?" paramsDesc={['期刊 id，可在对应期刊页 URL 中找到', '分类，见下表，默认为 All Recent Articles']} radar="1" rssbud="1" supportScihub="1">
+
+::: tip 提示
```

Comment:
```suggestion
:::tip
```
The way the tip title will be handled by docusarus' internal i18n instead of hardcoded string.

Reply:
👌